### PR TITLE
sql/internal/specutil: support input variables

### PR DESCRIPTION
**Pull request by rotemtam**

### CODE REVIEW

File: sql/internal/lang/vars.go [hunk 1]
```diff
@@ -0,0 +1,43 @@
+package lang
```
Comment (masseelch) [inline sql/internal/lang/vars.go:1]: license

File: sql/internal/lang/vars.go [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+				specutil.TypeSpec("int"),
+				specutil.TypeSpec("bool"),
+				specutil.TypeSpec("float"),
+			),
+		).Specs(),
+	))
+)
+
+func init() {
+	schemaspec.Register("variable", &Var{})
```
Comment (masseelch) [inline sql/internal/lang/vars.go:42]: Why not `var`?

**Review by masseelch — APPROVED** [review]
:blush:

File: sql/internal/lang/vars.go (hunk 1 above)
Comment (a8m) [inline sql/internal/lang/vars.go:1]: We agree to move it `sql/internal/specutil`.

File: sql/internal/lang/vars.go (hunk 2 above)
Comment (a8m) [reply sql/internal/lang/vars.go:42]: Because `terraform` uses `variables`. Maybe we should accept the 2 forms.

Comment (rotemtam) [reply sql/internal/lang/vars.go:42]: trying to keep things familiar for terraform users:
https://www.terraform.io/language/values/variables#declaring-an-input-variable

**Review by a8m — APPROVED** [review]

**Pull request merged**